insights/parsers/mongod_conf.py

```diff
@@ -3,10 +3,12 @@
 ==================================
 
 This module contains the following files:
-    ``/etc/mongod.conf`` and ``/etc/mongodb.conf``.
+    ``/etc/mongod.conf``,
+    ``/etc/mongodb.conf`` ,
+    ``etc/opt/rh/rh-mongodb26/mongod.conf``.
```

Comment:
Why remove `/` from `etc/opt/rh/rh-mongodb26/mongod.conf` but leave it in above two lines?